stdpopsim/ext/selection.py

```diff
@@ -16,11 +16,11 @@ class MutationType(object):
     # using rates obtained from the relative weights of the types.
     # I.e. the weights will be used in the ``proportion`` parameter
     # to SLiM's :func:`initializeGenomicElementType()`.
-    weight = attr.ib(default=0, type=float)
+    proportion = attr.ib(default=0, type=float)
 
     def __attrs_post_init__(self):
-        if self.weight < 0:
-            raise ValueError("weight must be >= 0.")
+        if self.proportion < 0 or self.proportion > 1:
+            raise ValueError("proportion must lie within the [0,1] interval.")
```

Comment:
I agree that proportion is a better name for this, to match the SLiM name. SLiM allows proprtions to be greater than 1 though. Do we want to allow that too?

Reply:
I think these should be strict proportions, because then we can use them to deal with the overlaying of neutral mutations. Alternatively, we could parametrize as mutation rates, but I like the idea of having one knob for overall mutation rate and knobs for proportions (of the total mutation rate) for each mutation type.